icu4c/source/test/cintltst/udatpg_test.c

```diff
@@ -836,4 +840,46 @@ static void TestRegionOverride(void) {
         udatpg_close(dtpg);
     }
 }
+
+static void TestISO8601(void) {
+    typedef struct TestCase {
+        const char* locale;
+        const UChar* skeleton;
+        const UChar* expectedPattern;
+    } TestCase;
+
+    const TestCase testCases[] = {
+        { "en_GB@calendar=iso8601;rg=uszzzz", u"EEEEyMMMMdjmm", u"y MMMM d, EEEE 'at' h:mm a" },
+        { "en_GB@calendar=iso8601;rg=uszzzz", u"EEEEyMMMMdHmm", u"y MMMM d, EEEE 'at' HH:mm" },
```

Comment:
@macchiati It doesn't seem like we should be getting the "at" here.  I think the `%atTime` patterns from CLDR didn't make it over to ICU properly.

Reply:
I'd still like some guidance on this issue.  I sent a private email.  This might actually be another instance of the problem described in https://unicode-org.atlassian.net/issues/ICU-22688 ...

Reply:
Mark and I discussed this offline; I think this is a bug in the CLDR-to-ICU tooling. I'm going to file a new ticket.